addons/sale/models/product_template.py

```diff
@@ -239,12 +240,13 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
             price = product.price if pricelist else list_price
             display_image = bool(product.image_1920)
             display_name = product.display_name
+            uom_count = product.uom_count
```

Comment:
uos_count is not define in sale :thinking:

Reply:
Yes, still moving things around

Reply:
This was resolved with the latest changes (can't mark it as resolved as I'm not the owner of the PR)